docs/external_dependencies.rst

```diff
@@ -51,15 +51,23 @@ To accomplish this BLT provides a ``DEPENDS_ON`` option for the
 The ``blt_register_library`` macro allows you to expose any external dependency,
 including any additional includes, libraries, compile flags, link flags, defines, etc that are required to use the dependency via a single name.
 
-For example, instead adding the include directories and libraries necessary to use a dependency *zzz* directly to your CMake executable target, if  *zzz* was
-registered with ``blt_register_library``, you can simply use 
+For example, instead adding the include directories, libraries, and other information
+necessary to use an external dependency *axom* directly to your CMake target,
```

Comment:
This sentence is a bit awkward.  (it was awkward in the original as well).

Reply:
should be better now.